charts/spire/charts/spire-agent/README.md

```diff
@@ -109,3 +109,6 @@ A Helm chart to install the SPIRE agent.
 | `experimental.enabled`                            | Allow configuration of experimental features                                                                                                                                                      | `false`                                                                          |
 | `experimental.syncInterval`                       | Sync interval with SPIRE server with exponential backoff                                                                                                                                          | `5s`                                                                             |
 | `experimental.featureFlags`                       | List of developer feature flags                                                                                                                                                                   | `[]`                                                                             |
+| `sockets.hostBasePath`                            | Path on which the agent socket is made available when admin.expose is true                                                                                                                        | `/run/spire/agent/sockets`                                                       |
```

Comment:
Should this comment be referring to `/tmp/spire-agent/private/admin.sock`?

Reply:
It needs to always be /tmp/spire-agent/private/admin.sock within the container so the cli can find it.

Outside the container, on the host, it should be configurable.

Reply:
I don't think there is a valid use case for putting the socket outside of the Pod.  The client that works with the socket is the `spire-agent` executable, called in a non-"run" context.